docs/workflows/aws.md

```diff
@@ -78,11 +79,14 @@ The AWS pipeline runs using a docker container. At runtime, the script [run_aws_
 - `burst_id_list` -> A list of burst id's corresponding to the scene. If not provided, all will be processed. Can be space separated list or line separated .txt file.
 - `resolution` -> The target resolution of the products. Default is 20m.
 - `output_crs` -> The target crs of the products. If not specified, the UTM of the scene center will be used. Expects integer values (e.g. 3031)
-- `dem` -> The dem to be used in processing. Supported is `cop_glo30`.
+- `dem_type` -> The dem to be used in processing. Supported is [`cop_glo30`, `REMA_32`, `REMA_10`, `REMA_2`].
```

Comment:
Suggest

The digital elevation model (DEM) to use for processing. Supported values: `cop_glo30`, `REMA_32`, `REMA_10`, `REMA_2`.